eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/FixedSizeMap.java

```diff
@@ -70,7 +70,12 @@
     @Override
     default FixedSizeMap<K, V> withMap(Map<? extends K, ? extends V> map)
     {
-        this.putAll(map);
-        return this;
```

Comment:
This is a breaking change albeit a correct change. But I would want it to be a part of a separate commit to show it on the Release notes.